bitnami/fluentd/templates/forwarder-init-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.forwarder.initScripts }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:  {{ printf "%s-init-scripts" (include "common.names.fullname" .) }}
```

Comment:
Both Aggregator and Forwarder configMap are called the same, this could cause issues if both were enabled at the same time.